mappings_test.go

```diff
@@ -255,7 +255,7 @@ func TestGetStringArrayOrNull(t *testing.T) {
 			name: "Test for invalid type",
 			args: args{
 				data: mappingTest,
-				key:  mappingIntArrayKey,
+				key:  mappingMapKey,
```

Comment:
Why did this need a change?

Reply:
`IntArray` can be converted to `StringArray`.
See
• https://github.com/PufferPanel/PufferPanel/blob/8a3b31c009b6ecb9bbda69ed1c4a5bb5eeb578f3/mappings.go#L90
• https://github.com/spf13/cast/blob/2b0eb0f724e320b655240e331aef36d1175986c2/caste.go#L1250-L1254

An example of an invalid type that the test expects is a map, hence I’m using it here.